tests/test_mllam_dataset.py

```diff
@@ -25,7 +27,8 @@
 )
 
 
-def test_retrieve_data_ewc():
+@pytest.fixture
```

Comment:
if you set the scope to "module" the fixture will only be run once for this test file, meaning we wont download the data again for each test. 

You can then also add the argument `meps_example_reduced_filepath` to the functions ` test_create_graph_reduced_meps_dataset` and `test_train_model_reduced_meps_dataset` to allow those to run independently.


```suggestion
@pytest.fixture(scope="module")
```